### ISSUE TITLE
kitchen-ec2 ssh port

### ISSUE THREAD

**Issue by pantzjantz** [test-kitchen/kitchen-ec2#396]
Maybe I'm just missing something but is there anyway to use an alternative ssh port? I'm attempting to use the ec2 driver to test my chef configuration of Github Enterprise (ami: ami-003f7be2a1c913a86). Since GHE uses port 22 for ssh clones it redirects ssh logins to port 122. When i run kitchen converge it just endlessly waits to login on port 22. I realize this is kind of a unique use-case but it would be nice to have. Thanks --chris

**Comment by pantzjantz** [comment]
Awesome! That's looking like its going to work.. although I just realized where I'm trying to come from port 122 is blocked by the firewall. I can see that its now trying to use port 122 rather than 22 so once I get the firewall sorted I should be good. Thanks for the quick reply!

Waiting for SSH service on 10.X.X.X:**122**, retrying in 3 seconds
